discovery-provider/plugins/notifications/src/setup.ts

```diff
@@ -19,26 +19,26 @@ export async function setupTriggers(db: Knex) {
     .where('routine_name', '=', functionName)
     .count({ count: '*' })
 
-  let skip = parseInt(count) == 1
-
-  if (skip) {
-    logger.info(`function ${functionName} already exists... skipping`)
-  } else {
-    // drop existing triggers
-    logger.info(`dropping any existing triggers`)
-    await db.raw(`drop trigger if exists trg_notification on notification;`)
-
-    // create function
-    logger.info(`creating plpgsql function`)
-    await db.raw(trigger)
-
-    // create triggers
-    logger.info(`creating triggers`)
-    if (process.argv[2] !== 'drop') {
-      await db.raw(`
+  // let skip = parseInt(count) == 1
+
+  // if (skip) {
+  // logger.info(`function ${functionName} already exists... skipping`)
+  // } else {
+  // drop existing triggers
```

Comment:
can we remove these

Reply:
and same for the commented lines 37 and 42-43

Reply:
This was commented out for testing, will undo, thx